server/tools/patch/tool.go

```
@@ -124,14 +131,21 @@ func (t *Tool) targetsForeignManagedWorktree(doc patchformat.Document) (bool, er
 			if err != nil {
 				return false, err
 			}
-			if t.managedWorktreePathContext.IsForeignManagedWorktreePath(path, resolved) {
+			if err := t.checkForeignManagedWorktreePath(resolved); err != nil {
 				return true, nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 134) but it returns nil

(nilerr)